### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by reilem** [aptabase/aptabase_flutter#3]
I was forced to downgrade `package_info_plus` and `device_info_plus` when installing `aptabase_flutter`, and as a result also my `http` dependency.

It would be nice if you guys could use the latest version of these packages so that users don't have to downgrade.

- Upgrade `package_info_plus` to latest version 4.0.1: https://pub.dev/packages/package_info_plus/versions
- Upgrade `device_info_plus` to latest version 9.0.3: https://pub.dev/packages/device_info_plus

As far as I can tell there were no breaking changes between the versions so hopefully an upgrade should be pretty trivial. If you guys don't have time I can maybe open a PR for this change.

**Comment by goenning** [comment]
I agree, we can simply bump the deps. Would you like to submit a PR?

**Comment by reilem** [comment]
Yep! Everything looks to be working. Thanks 👍
